Description: Remove most print and sleep.
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 grub2 (2.02+dfsg1+deepin3-1deepin13) unstable; urgency=medium
 .
   * Install signed efi from monolithic directory.
Author: Zhang Yueqian <zhangyueqian@deepin.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2019-12-18

--- grub2-2.02+dfsg1+deepin3.orig/grub-core/commands/verify.c
+++ grub2-2.02+dfsg1+deepin3/grub-core/commands/verify.c
@@ -959,12 +959,6 @@ static grub_ssize_t
 pseudo_read (struct grub_file *file, char *buf, grub_size_t len)
 {
   grub_memcpy (buf, (grub_uint8_t *) file->data + file->offset, len);
-  grub_dprintf ("sbverify", "reading file of data: HEAD %X%X%X%X, END %X%X%X%X\n",
-		(unsigned int) buf[0], (unsigned int) buf[1],
-		(unsigned int) buf[2], (unsigned int) buf[3],
-		(unsigned int) buf[len-4], (unsigned int) buf[len-3],
-		(unsigned int) buf[len-2], (unsigned int) buf[len-1]);
-  grub_millisleep (8000);
   return len;
 }
 
@@ -1012,8 +1006,6 @@ GRUB_MOD_INIT(verify)
     key_count += 1;
     grub_printf ("sbverify: loading keys of %d\n", key_count);
 
-    grub_millisleep (1000);
-
     pseudo_file.fs = &pseudo_fs;
     pseudo_file.size = (header->size - sizeof (struct grub_module_header));
     pseudo_file.data = (char *) header + sizeof (struct grub_module_header);
@@ -1026,8 +1018,6 @@ GRUB_MOD_INIT(verify)
     grub_pk_trusted = pk;
   }
 
-  grub_millisleep (10000);
-
   /*
    * do not force enable check_signatures when import public keys.
   if (!val)
--- grub2-2.02+dfsg1+deepin3.orig/grub-core/loader/i386/linux.c
+++ grub2-2.02+dfsg1+deepin3/grub-core/loader/i386/linux.c
@@ -784,12 +784,13 @@ grub_cmd_linux (grub_command_t cmd __att
 	      grub_dprintf ("sbverify", "verify success!\n");
 	    }
 	  else
-	    grub_dprintf ("sbverify", "Error: %d\n", (int) err);
+	    {
+	      grub_error (err, "File %s verify failed!\n", argv[0]);
+	      goto fail;
+	    }
 	}
     }
 
-  grub_millisleep (30000);
-
   if (grub_file_read (file, &lh, sizeof (lh)) != sizeof (lh))
     {
       if (!grub_errno)
